app/src/main/res/drawable/ic_media_library.xml

```diff
@@ -0,0 +1,9 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Please auto-format this file: It should be the same as ic_file_hidden.xml in terms of identation.